cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/ElasticsearchSinkConsumerIT.java

```diff
@@ -86,13 +87,13 @@ public static void beforeAll() throws Exception {
         containerNetwork = Network.newNetwork();
 
         kafkaContainer = new KafkaContainer(DockerImageName.parse(KAFKA_IMAGE))
-                            .withNetworkAliases("kafka")
-                            .withNetwork(containerNetwork);
+                .withNetworkAliases("kafka")
```

Comment:
Lost of formatting changes? Did we use new formatter?

Reply:
If there is one project level or mvn based format plugin, we should use that.

Reply:
There is one mvn based format plugin, this formatting was a result of using that only. I will revert these changes and will take them up in the other refactoring PR I will raise.